### DISCUSSION TITLE
Nillion node

### DISCUSSION THREAD

**Discussion by karandedhaa** [community/community#137432] [Programming Help]
### Nillion node

General

### Body

@Alexjptz I'm currently running nillion node and my secrets are increasing but my accuser txn are showing in explorer though it's generating txn and some times it shows error account mismatch

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Karan10-cyber ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
